### PR TITLE
PLANET-7677: Consolidate parent act page injected to window p4_vars

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-7677

In preparation of https://jira.greenpeace.org/browse/PLANET-7657

# Description
Remove appearences of p4ge_vars to avoid conflicts whenever the plugin is disabled.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [umbriel](https://www-dev.greenpeace.org/test-umbriel) | [admin](https://www-dev.greenpeace.org/test-umbriel/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-umbriel/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-umbriel) | [composer-local.json](https://github.com/greenpeace/planet4-test-umbriel/blob/main/composer-local.json)

:watch: 2024.11.26 13:50:11

File: assets/src/blocks/ActionsList/index.js [hunk 1]
```diff
@@ -9,8 +9,7 @@ export const registerActionsListBlock = () => {
   const {registerBlockVariation} = wp.blocks;
   const {__} = wp.i18n;
 
-  const IS_NEW_IA = window.p4_vars.options.new_ia === 'on' ||
-    window.p4_vars.options.new_ia === true;
+  const IS_NEW_IA = window.p4_vars.options.new_ia === 'on' && true;
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/ActionsList/index.js:12]: Change to `const IS_NEW_IA = window.p4_vars.options.new_ia === 'on'`

File: src/MasterBlocks.php [hunk 2]
```diff
@@ -173,6 +173,7 @@ public function enqueue_block_public_assets(): void
     private function get_p4_options(): array
     {
         $option_values = get_option('planet4_options');
+        $is_new_ia = $option_values['new_ia'] === 'on' ? true : false;
```
Comment (mleray) [inline src/MasterBlocks.php:176]: Isn't `$is_new_ia = $option_values['new_ia'] === 'on';` enough? It's already a boolean

File: assets/src/blocks/ActionsList/index.js [hunk 3]
```diff
@@ -9,8 +9,7 @@ export const registerActionsListBlock = () => {
   const {registerBlockVariation} = wp.blocks;
   const {__} = wp.i18n;
 
-  const IS_NEW_IA = window.p4_vars.options.new_ia === 'on' ||
-    window.p4_vars.options.new_ia === true;
+  const IS_NEW_IA = window.p4_vars.options.new_ia;
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/ActionsList/index.js:12]: Already a boolean https://github.com/greenpeace/planet4-master-theme/pull/2455/files#diff-6b8792fb39e5bc086df0f352486bd1b87cd573422d0702519e6f77f0103c7ef3R193

File: src/MasterBlocks.php [hunk 4]
```diff
@@ -173,6 +173,7 @@ public function enqueue_block_public_assets(): void
     private function get_p4_options(): array
     {
         $option_values = get_option('planet4_options');
+        $is_new_ia = $option_values['new_ia'] === 'on' ;
```
Comment (mleray) [inline src/MasterBlocks.php:176]: I see that in other places in the code we use [this check](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Settings.php#L62), maybe we should be consistent and use this here too?

**Pull request merged**